docs/architecture/resilience.txt

```diff
@@ -114,6 +114,15 @@ Here are some considerations:
    rare consistency or data-loss issues you might encounter when CrateDB
    experiences network or hardware failures (see next section).
 
+
+Nightly Jepsen tests for dirty reads, lost updates, version divergence
+----------------------------------------------------------------------
+
+We run nightly tests of the Jepsen suite which test dirty reads, lost updates, and version divergence.
+The source code of these tests can be inspected in the official `Jepsen repository <https://github.com/jepsen-io/jepsen>`_.
+The tests are passing.
```

Comment:
that is not true, they are currently expected to fail which they do

Reply:
I know we don't support all Jepsen tests but I thought the subset described here passes. Also the latest run passes: https://jenkins.crate.io/job/CrateDB/job/crate_jepsen_test/229/ The jepsen.log files in the artifact doesn't report any failure.

Reply:
This job will pass even if the jepsen tests are failing (see https://jenkins.crate.io/job/CrateDB/job/crate_jepsen_test/200). Its not that easy to let them fail in a deterministic way. The main intention of this job is to ensure that our jepsen suite and tests are running without any issues against the current crate version despite of the test results.